src/Altinn.App.Core/Configuration/GeneralSettings.cs

```diff
@@ -34,6 +34,11 @@ public class GeneralSettings
     /// </summary>
     public string HostName { get; set; } = "local.altinn.cloud";
 
+    // Only here to be overridden from tests
+    internal string LocaltestUrl { get; set; } = "http://local.altinn.cloud";
```

Comment:
How is `GeneralSettings.LocaltestUrl` used in relation to `LocaltestValidation.ExpectedHostname`. I don't think you ment to use both.

Reply:
`LocaltestUrl` was introduced to support overriding the HTTP client request URL, so that's the only use of it. `ExpectedHostname` is only used to check `HostName` (for when to run the code in the hosted service). There is a test for simulating DNS failure where `LocaltestUrl` differs from `HostName` (if I changed `HostName` the code wouldn't run). I found this confusing myself too tbh 😅

Reply:
In the latest commit I removed this field and used `PlatformSettings` instead for that purpose. I think that is what you proposed in the other comment?

Reply:
Yes.